### PR TITLE
Toasts

### PR DESCRIPTION
**Pull request by dev-enforced**
In this PR you will find the toasts component provided by this component library which happens on the click of a button and in the documentation preview you can find the buttons required to see the toast. The code snippet also has the HTML structure with the required classes required to make a toast.

Here is the live Preview link for the toasts component: https://deploy-preview-15--gentle-ui.netlify.app/documentation/toasts.html

### CODE REVIEW

File: documentation/toasts.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            } else if (snackbarType === "LEADING") {
+                leadingSnackbar.style.display = "flex";
+                setTimeout(() => {
+                    leadingSnackbar.style.display = "none";
+                }, 2000);
+            } else if (snackbarType === "STACKED") {
+                stackedSnackbar.style.display = "flex";
+                setTimeout(() => {
+                    stackedSnackbar.style.display = "none";
+                }, 2000);
```
Comment (fahd23) [inline documentation/toasts.html]: One Suggestion, good to have scripts in the separate file

**Review by fahd23 — APPROVED** [review]
Given Suggestion, Code looks clean

File: components/toasts.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  font-family: "Roboto-slab", sans-serif;
+}
+.snackbar-actions {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+}
+.gentle-snackbar-link {
+  text-decoration: none;
+  color: #af8ddd;
```
Comment (Namrata-J) [inline components/toasts.css]: you could try using color variables here

File: components/toasts.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  border-radius: 40px;
```
Comment (Namrata-J) [inline components/toasts.css]: would suggest rems for size greater than 10px

File: components/toasts.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  display: none;
+  flex-direction: column;
+  position: fixed;
+  bottom: 10%;
+  gap: 1rem;
+}
+
+.close-icon {
+  color: var(--text-color);
+  border-radius: 50px;
```
Comment (Namrata-J) [inline components/toasts.css]: change it here as well

**Review by Namrata-J — APPROVED** [review]

File: documentation/toasts.html (hunk 1 above)
Comment (dev-enforced) [reply documentation/toasts.html]: Yes @fahd23 all scripts have been moved to a common folder

File: components/toasts.css (hunk 4 above)
Comment (dev-enforced) [reply components/toasts.css]: Done [b7eeec6](https://github.com/dev-enforced/Gentle_UI/pull/15/commits/b7eeec6ad460d8060f7c1dc50eb7495926fb5647)

File: components/toasts.css (hunk 3 above)
Comment (dev-enforced) [reply components/toasts.css]: Done [b7eeec6](https://github.com/dev-enforced/Gentle_UI/pull/15/commits/b7eeec6ad460d8060f7c1dc50eb7495926fb5647)

File: components/toasts.css (hunk 2 above)
Comment (dev-enforced) [reply components/toasts.css]: Done [b7eeec6](https://github.com/dev-enforced/Gentle_UI/pull/15/commits/b7eeec6ad460d8060f7c1dc50eb7495926fb5647)

File: documentation/toasts.html (hunk 1 above)
Comment (dev-enforced) [reply documentation/toasts.html]: Here [36f6c49](https://github.com/dev-enforced/Gentle_UI/pull/15/commits/36f6c49a14b73c836aea7987077afe09aa911c6b)

**Pull request merged**